src/components/editgrid/EditGrid.js

```diff
@@ -148,13 +140,13 @@ export default class EditGridComponent extends NestedComponent {
               }),
               this.ce('div', {class: 'editgrid-actions'},
                 [
-                  this.ce('div', {
+                  this.ce('button', {
```

Comment:
Why are we changing these to a button? If we do then we need to use event.preventDefault and stop propogation.